tajo-cli/src/main/java/org/apache/tajo/cli/tools/TajoDump.java

```diff
@@ -216,6 +217,8 @@ private static void dumpDatabase(TajoClient client, String databaseName, PrintWr
         System.err.println("ERROR:" + tableName + "," + e.getMessage());
       }
     }
+
+    LOG.warn("Partition dump and restore are not supported yet");
```

Comment:
I had misunderstood your comments and added a printed message and unit test cases.